Makefile.toml

```diff
@@ -600,6 +600,30 @@ docker run --rm \
           | jq -r '.'
 '''
 
+[tasks.update-workspace-dependency-versions]
+dependencies = ["install-toml-cli"]
+script = '''
+next_version=$1
+
+workspace_dependencies=( \
+  axone-cognitarium \
+  axone-rdf \
+  axone-wasm \
+  axone-cognitarium-client \
+  axone-logic-bindings \
+  axone-objectarium \
+  axone-objectarium-client \
+)
+
+for workspace_dependency in ${workspace_dependencies[@]}; do
+  toml set Cargo.toml workspace.dependencies.$workspace_dependency.version $next_version > Cargo.toml.tmp && mv Cargo.toml.tmp Cargo.toml
+done
+'''
+
+[tasks.publish-crates]
+dependencies = ["install-cargo-workspaces"]
+script = "cargo workspaces publish --publish-as-is --allow-branch main"
```

Comment:
Since all version as already been bumped through releaserc previous steps in a release process, I'm wondering if you need to pass `--from-git` argument avoiding execute `cargo workspaces version` before publish, accordingly to the documentation ? (https://github.com/pksunkara/cargo-workspaces#publish)

> Publish all the crates from the workspace in the correct order according to the dependencies. By default, this command runs [version](https://github.com/axone-protocol/contracts/pull/569/files#version) first. If you do not want that to happen, you can supply the --from-git option.

Also, you don't need to add `--allow-branch main` because after some check, the help command say that the default branch is `master` but an issue has been resolved to allow both `master` and `main` branch by default (it's has not been added in the documentation) : https://github.com/pksunkara/cargo-workspaces/issues/32 (https://github.com/pksunkara/cargo-workspaces/blob/master/cargo-workspaces/src/utils/git.rs#L139)

Reply:
> Since all version as already been bumped through releaserc previous steps in a release process, I'm wondering if you need to pass --from-git argument avoiding execute cargo workspaces version before publish, accordingly to the documentation ? (https://github.com/pksunkara/cargo-workspaces#publish)

I saw this but the option `--publish-as-is` is supposed to do this as well. 

> Also, you don't need to add --allow-branch main because after some check, the help command say that the default branch is master but an issue has been resolved to allow both master and main branch by default (it's has not been added in the documentation) : https://github.com/pksunkara/cargo-workspaces/issues/32 (https://github.com/pksunkara/cargo-workspaces/blob/master/cargo-workspaces/src/utils/git.rs#L139)

Ah cool I didn't see this